mungegithub/mungers/blunderbuss.go

```diff
@@ -57,15 +68,41 @@ func (b *BlunderbussMunger) RequiredFeatures() []string { return []string{featur
 // Initialize will initialize the munger
 func (b *BlunderbussMunger) Initialize(config *github.Config, features *features.Features) error {
 	b.features = features
+	b.aliasReadFunc = func() ([]byte, error) {
+		return ioutil.ReadFile(b.AliasFile)
+	}
 	return nil
 }
 
 // EachLoop is called at the start of every munge loop
-func (b *BlunderbussMunger) EachLoop() error { return nil }
+func (b *BlunderbussMunger) EachLoop() error {
+	if b.AliasFile == "" {
+		// no alias file specified, exit immediately.
+		return nil
+	}
+
+	// read and check the alias-file.
+	fileContents, err := b.aliasReadFunc()
+	if err != nil {
+		return fmt.Errorf("Unable to read alias file: %v", err)
+	}
+
+	hash := mungerutil.GetHash(fileContents)
+	if b.prevHash != hash {
+		structuredYaml := map[string]map[string][]string{}
+		if err := yaml.NewYAMLToJSONDecoder(bytes.NewReader(fileContents)).Decode(&structuredYaml); err != nil {
```

Comment:
Better to define a golang struct and parse it into that so we get compile time checks.